build-common/src/org/jetbrains/kotlin/build/report/ICReporter.kt

```diff
@@ -9,18 +9,27 @@ import org.jetbrains.kotlin.cli.common.ExitCode
 import java.io.File
 
 interface ICReporter {
-    fun report(message: () -> String)
-    fun reportVerbose(message: () -> String)
+
+    enum class ReportSeverity(val level: Int) {
+        WARNING(3),
+        INFO(2),
+        DEBUG(1);
+    }
+
+    fun report(message: () -> String, severity: ReportSeverity)
```

Comment:
This breaks `MavenICReporter` - could you fix it?

Reply:
I'm not familiar with `kotlin-maven-plugin` as it is not a Gradle project. I get unresolved symbols if I open that project directly. Do you know if there is a command/test I could run from the command line to confirm that everything is okay in that module after I fix the issue?

Reply:
try `mvn deploy` (there is also `mvnw` [tool](https://maven.apache.org/wrapper/) available, but you need to add it yourself)

Reply:
Thanks, I've managed to work with that project. I've fixed this in the top commit.